src/ILLink.Tasks/LinkTask.cs

```diff
@@ -415,16 +415,24 @@ protected override string GenerateResponseFileCommands ()
 				}
 			}
 
+			bool debugger_supported = true;
+
 			if (FeatureSettings != null) {
 				foreach (var featureSetting in FeatureSettings) {
 					var feature = featureSetting.ItemSpec;
 					var featureValue = featureSetting.GetMetadata ("Value");
 					if (String.IsNullOrEmpty (featureValue))
 						throw new ArgumentException ("feature settings require \"Value\" metadata");
 					args.Append ("--feature ").Append (feature).Append (' ').AppendLine (featureValue);
+
+					if (feature == "System.Diagnostics.Debugger.IsSupported" && bool.TryParse (featureValue, out bool fValue))
```

Comment:
I don't think the task is the right place for logic like this - the task should just translate options from MSBuild language into command-line args. Instead this should either be built-in to the linker, or moved to MSBulid targets.

Reply:
Replaces with explicit task setting